multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/NamespaceGlobalParameters.java

```diff
@@ -20,26 +20,30 @@ public NamespaceGlobalParameters(DeploymentDescriptor deploymentDescriptor) {
     }
 
     public boolean getApplyNamespaceAppNamesParameter() {
-        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_APPS);
+        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_APPS, Boolean.TRUE);
     }
 
     public boolean getApplyNamespaceServiceNamesParameter() {
-        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_SERVICES);
+        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_SERVICES, Boolean.TRUE);
     }
 
     public boolean getApplyNamespaceAppRoutesParameter() {
-        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_ROUTES);
+        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_ROUTES, Boolean.TRUE);
+
+    }
 
+    public boolean getApplyNamespaceAsSuffix() {
+        return getApplyNamespace(SupportedParameters.APPLY_NAMESPACE_AS_SUFFIX, Boolean.FALSE);
     }
 
-    private boolean getApplyNamespace(String applyNamespaceFlagName) {
+    private boolean getApplyNamespace(String applyNamespaceFlagName, boolean defaultValue) {
         if (!applyNamespaceGlobalLevel.isEmpty()) {
             Object value = applyNamespaceGlobalLevel.get(applyNamespaceFlagName);
             if (value != null && !(value instanceof Boolean)) {
                 throw new ContentException(NAMESPACE_PARSING_ERROR_MESSAGE, applyNamespaceFlagName);
             }
             return (boolean) applyNamespaceGlobalLevel.getOrDefault(applyNamespaceFlagName, true);
```

Comment:
if APPLY_NAMESPACE_AS_SUFFIX exists at the global level but is 'nulll' this will return 'true' as opposed to the 'false' default value given below - is this correct behaviour?